test/sqllogictest/materialized_views.slt

```diff
@@ -1619,3 +1619,118 @@ create  cluster-replica  "c_schedule_1"  "schedule"  false  {"decision":"on","ob
 create  cluster-replica  "c_schedule_3"  "schedule"  false  {"decision":"on","objects_needing_refresh":["uXXX"],"rehydration_time_estimate":"00:00:00"}
 create  cluster-replica  "c_schedule_4"  "schedule"  false  {"decision":"on","objects_needing_refresh":["uXXX"],"rehydration_time_estimate":"00:00:00"}
 create  cluster-replica  "c_schedule_rehydration_time_estimate"  "schedule"  false  {"decision":"on","objects_needing_refresh":["uXXX"],"rehydration_time_estimate":"00:16:35"}
+
+# Materialized views in this file can be explained
+
+simple conn=mz_system,user=mz_system
+ALTER SYSTEM SET enable_explain_pushdown = true
+----
+COMPLETE 0
+
+# Pulling stats for refresh-every and similar MVs can time out,
+# since data may not yet be available...
+
+simple conn=mz_system,user=mz_system
+ALTER SYSTEM SET statement_timeout = '1s'
```

Comment:
This seems to be not taking effect. If you run the slt with `-vv`, which prints every statement, it's visible that the last 3 statements each take about 10 seconds, which is the default statement timeout. Maybe this `ALTER` doesn't affect existing sessions? There is a notice that kind of suggests this:
```
materialize=> ALTER SYSTEM SET statement_timeout = 1;
NOTICE:  variable "statement_timeout" was updated for the system, this will have no effect on the current session
ALTER SYSTEM
```
Although this talks about the current session, maybe it's also true for any other ongoing session?

@MaterializeInc/testing , is there a way to reliably change the statement timeout in an slt?

Also, should we change the message of this notice if the reality is that indeed any other ongoing session is not affected?

(Alternatively, you could just not bother with the statement timeout, but explain only one of these MVs where the explain will block.)

Reply:
Not sure, could maybe testdrive's `$ set-sql-timeout` help you?

Reply:
I guess that could work. So, instead of this slt, you could test blocking explains in `materialized-view-refresh-options.td`, where e.g. `mv9` and `mv10` should also be non-readable, similarly to these MVs.

Reply:
Ah! This should be `SET statement_timeout = '1s'`, not `ALTER SYSTEM SET statement_timeout = 1;`, if we want it to just apply to the current session.

I've pushed a commit with the proper syntax. I think this addresses your first comment too... since it's session local there's no need to unset it.

> If you run the slt with `-vv`

I was not aware of this option... thanks for mentioning! Will save me some time in the future.